lib/charms/vault_k8s/v0/vault_client.py

```diff
@@ -178,6 +178,8 @@ def enable_audit_device(self, device_type: AuditDeviceType, path: str) -> None:
             )
             logger.info("Enabled audit device %s for path %s", device_type.value, path)
         except InvalidRequest as e:
+            if not e.json:
+                raise VaultClientError(e) from e
```

Comment:
didn't know you could do `from` like this. Why wasn't any of this caught by tox before?

Reply:
The `from` style has nothing to do with tox. It didn't catch that, that is a change I made last week.

Unrelated, tox wasn't checking any of these files because calling `pyright` without any parameters just checks `src/` for some reason.